pkg/tekton/crds.go

```diff
@@ -124,6 +125,30 @@ func (crds *CRDWrapper) ObjectReferences() []kube.ObjectReference {
 	return resources
 }
 
+func (crds *CRDWrapper) String() string {
+	var allResources []interface{}
+	allResources = append(allResources, crds.pipeline)
+	for _, task := range crds.tasks {
+		allResources = append(allResources, task)
```

Comment:
You can replace the entire `for` with `append(allResources, task...).

Reply:
Now I remember, I tried this and it did not work. I was surprised as well. It gives you a compile error:
```
pkg/tekton/crds.go:131:23: cannot use crds.tasks (type []*"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1".Task) as type []interface {} in append
```